cdk/script/ci

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+source ${DIR}/setup
+source ${DIR}/test
+
+for file in bin/*; do
```

Comment:
With only one file in the `bin` directory, we can drop the `for` I think (the CLI needs updating!)

Reply:
I've updated to drop for now, but suspect will reintroduce when the follow on PR hits, which will also introduce a stack for the VPC.

Reply:
Ah, forgot about that. Sorry for the pain!